docs/docker.rst

```diff
@@ -0,0 +1,162 @@
+Running Rally with Docker
+=========================
+
+Rally is also available as a `Docker image <https://hub.docker.com/r/elastic/rally>`_.
```

Comment:
Remove the `also`?

Reply:
Addressed in https://github.com/elastic/rally/pull/702/commits/f2e6db89dbd9cde0414581dbfe4a83a808013aa9